### DISCUSSION TITLE
How to display Thread Products on the website(openthread.io)

### DISCUSSION THREAD

**Discussion by fuxiaoming-lumi** [openthread/openthread#8352] [Q&A] [answered]
Our Thread product has obtained the Thread certification and is developed based on **openthread**. We also want to display these products on the [openthread.io](openthread.io). But I don't know what the specific process should be. Any help would be deeply appreciated.

**Comment by jwhui** [comment, marked as answer]
You can send email directly to me (jonhui@google.com). Thanks!
